Zappy_IA_Report/DTC/Code/models.py

```
@@ -0,0 +1,16 @@
+class iaParser:
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)